CMIOMinimalSample/Stream.mm

```diff
@@ -171,22 +171,16 @@ - (void)fillFrame {
 
     CVPixelBufferRef pixelBuffer = [self createPixelBufferWithTestAnimation];
 
-    // The timing here is quite important. For frames to be delivered correctly and successfully be recorded by apps
-    // like QuickTime Player, we need to be accurate in both our timestamps _and_ have a sensible scale. Using large
-    // timestamps and scales like mach_absolute_time() and NSEC_PER_SEC will work for display, but will error out
-    // when trying to record.
-    //
-    // Instead, we start our presentation times from zero (using the sequence number as a base), and use a scale that's
-    // a multiple of our framerate. This has been observed in parts of AVFoundation and lets us be frame-accurate even
-    // on non-round framerates (i.e., we can use a scale of 2997 for 29,97 fps content if we want to).
-    CMTimeScale scale = FPS * 100;
-    CMTime frameDuration = CMTimeMake(scale / FPS, scale);
-    CMTime pts = CMTimeMake(frameDuration.value * self.sequenceNumber, scale);
+    double hostTime = CACurrentMediaTime();
+    double timescale = 600.0;
+
+    CMTime eventTime = CMTimeMake(hostTime*timescale,timescale);
```

Comment:
I think @iKenndac's solution in #12 is little better (using `frameDuration.value * self.sequenceNumber` for the timestamp) since it won't be affected by the timing when the system is able to execute this code. The timestamp shall be the timing when the frame _should_ be presented, not the timing when the OS happened to get back to this thread.